decidim-core/lib/decidim/core/engine.rb

```diff
@@ -88,6 +88,7 @@ class Engine < ::Rails::Engine
       end
 
       initializer "decidim.graphql_api" do
+        GraphQL::DeprecatedDSL.activate
```

Comment:
```suggestion
        # Enable them method `!` everywhere for compatibility, this line will be removed when upgrading to GraphQL 2.0
        GraphQL::DeprecatedDSL.activate
```
Maybe we should add a comment here to remind a future removal